src/App.jsx

```diff
@@ -3,18 +3,20 @@ import React, {useState, useEffect} from "react";
 export const App = () => {
 
     const [value, setValue] = useState('')
-    const [responseData, setResponseData] = useState('')
+    const [responseData, setResponseData] = useState([])
 
-    useEffect(async () => {
-        const response = await fetch('http://localhost:8008/albums')
-        const data = await response.json()
-        setResponseData(data)
-    })
+    useEffect(() => {
+        fetch(`http://localhost:8008/value=${value}`)
+        .then(response => response.json())
```

Comment:
Why 'then'? Use await instead.

Reply:
Because I get an error, that you can't use async function i useEffect :)

Reply:
*in useEffect

Reply:
Alright